### PR TITLE
Added data for Redfin

### LINKED ISSUE #53: Add data for Redfin
#### Description

Add a file with data for Redfin.  They may not be on the Fortune 500.

Here are 2 of their statements:
https://twitter.com/Redfin/status/1271141352927969281?s=20
https://twitter.com/Redfin/status/1267211947973283840?s=20

Use `companies-on-blm/examples/add-a-company.md` as a reference for how to add data for a company. If information for a field cannot be found, leave it blank or delete that line.

### PR DESCRIPTION
**Pull request by Nyette**
# Description

Added data for Redfin.

Fixes #53 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

**Test Configuration**:
* Firmware version:
* Hardware:
* Toolchain:
* SDK:

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: companies/redfin.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+name: Redfin
+slug: redfin
+blm_statements:
+  - url: https://twitter.com/Redfin/status/1267211947973283840?s=20
+    date_posted: 2020-05-31
+  - url: https://twitter.com/Redfin/status/1271141352927969281?s=20
+    date_posted: 2020-06-11
+exec_team_url: [Leadership](https://press.redfin.com/spokespeople)
```
Comment (kelseyhuse30) [inline companies/redfin.md:9]:
```suggestion
exec_team_url: https://press.redfin.com/spokespeople
```

Just the URL here is fine!  We aren't sure where we are displaying this info quite yet.

File: companies/redfin.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+diversity_inclusion_url: [Diversity at Redfin in 2020](https://www.redfin.com/news/diversity-at-redfin-in-2020/)
```
Comment (kelseyhuse30) [inline companies/redfin.md:10]:
```suggestion
diversity_inclusion_url: https://www.redfin.com/news/diversity-at-redfin-in-2020/
```

**Review by kelseyhuse30 — CHANGES_REQUESTED** [review]
Thank you so much! Just a few updates from me. Love that you used Markdown - but for the front-matter data we prefer just the raw text :)

**Comment by Nyette** [comment]
Glad I could help! I'm happy to meet another Flatiron School graduate in the wild.

File: companies/redfin.md (hunk 1 above)
Comment (Nyette) [reply companies/redfin.md]: I made the necessary changes

**Review by kelseyhuse30 — APPROVED** [review]

**Comment by kelseyhuse30** [comment]
@Nyette no way!!! Haha that's awesome.  We are everywhere :D

**Pull request merged**